<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cerbere</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="home">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Bienvenue sur notre nouvelle application - Cerbere</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="port_scan">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Port scan</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="ip_label">
            <property name="text">
             <string>Ip Addresse</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="start_label">
            <property name="text">
             <string>Start Port</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="end_label">
            <property name="text">
             <string>End Port</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="ip_input"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="start_input"/>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="end_input"/>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="scan_btn">
            <property name="text">
             <string>Scan</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <column>
           <property name="text">
            <string>Port</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Status</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ip_scan">
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>190</y>
          <width>151</width>
          <height>111</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Ip Scan</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="terminal">
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>210</y>
          <width>291</width>
          <height>81</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Terminal</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuSave_as">
     <property name="title">
      <string>Save as</string>
     </property>
     <addaction name="actionJson"/>
     <addaction name="actionPdf"/>
    </widget>
    <addaction name="menuSave_as"/>
   </widget>
   <widget class="QMenu" name="menuUtilities">
    <property name="title">
     <string>Utilities</string>
    </property>
    <addaction name="actionPorts_scan"/>
    <addaction name="actionIp_scan"/>
    <addaction name="actionTerminal"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuUtilities"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionJson">
   <property name="text">
    <string>Json</string>
   </property>
  </action>
  <action name="actionPdf">
   <property name="text">
    <string>Pdf</string>
   </property>
  </action>
  <action name="actionPorts_scan">
   <property name="text">
    <string>Ports scan</string>
   </property>
  </action>
  <action name="actionIp_scan">
   <property name="text">
    <string>Ip scan</string>
   </property>
  </action>
  <action name="actionTerminal">
   <property name="text">
    <string>Terminal</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
